README.md

```diff
@@ -150,6 +129,79 @@ If CallKit has been set up correctly, Tom's device will ring like a phone call,
 
 Note: Due to an issue with ARKit, you cannot start a Remote Streem from a view using the camera https://forums.developer.apple.com/message/411888#411888
 
+#### Roles
+
+Roles dictate which side of the streem your on as in whether you're sharing your video or receiving video. They also affect which tools are available to you. The different roles are:
+
+* `LOCAL_CUSTOMER`: This is the primary role for the customer in a two-way streem
+* `ONSITE_CUSTOMER`: This is the primary role for a customer in an onsite (one-way) streem 
+* `REMOTE_PRO`: This is the primary role for the pro in a two-way streem
+* `ONSITE_PRO`: This is the primary role for a pro in an onsite (one-way) streem
```

Comment:
Why the word `primary` in each of these? Could we omit `This is the primary role for` in each?

Reply:
I'd suggest capitalizing `Customer` and `Pro` throughout.

Reply:
done and done